convene-web/app/views/rooms/edit.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="room-setting">
+  <h2>Configure <%= room.name %></h2>
+
+  <div class="room-setting__setting-card">
+    <%= form_with model: room, url: workspace_room_path(room.workspace, room), local: true do |room_form| %>
+      <fieldset>
+        <legend>Privacy and Security</legend>
+        <div class="room-setting__form-element-container">
```

Comment:
Not Blocking: It still feels odd to me that we have `form-element-container` classes that are _unique_ to a particular form when we literally only have 2 forms in the entire application. I'd rather we had a `form-element` class that plays nicely across most of our forms; and created exceptions only when absolutely necessary; and when we create those exceptions to do so by leveraging modifiers, like `--centered` or `--inline` or the block hierarchy `.form.update-room .privacy-and-security-settings .form-element { color: purple }`

Reply:
I think I was used to naming classes like Bootstrap framework and keep naming things container. `-container` can be removed.

Reply:
Apologies, to be more clear this feedback is not about the `container` suffix, and is more about how we use CSS selectors and class names to target parts of the HTML. For example, `.room-setting__form-element` vs `.form .--update .--room .form-element`